app/assets/javascripts/comments.js

```diff
@@ -240,3 +253,14 @@ $(() => {
     }
   });
 });
+
+
+window.new_thread = function (post_id){
```

Comment:
Avoid adding things to the `window` scope if at all possible. Look at where this function is used - is it only used in this file? If so, then you can just define it for this file. If you need it across multiple files, it'd be better to add this as an API method to `qpixel.js`, where it can come under the `window.QPixel` scope.

Function and parameter names should use `camelCase`, and ES6 function definitions where possible:

```suggestion
const newThread = (postId) => {
```